### PR TITLE
WIP: Add RetireJS to HuskyCI

### LINKED ISSUE #127: Add a RetireJS security Test
Let's now check JS code! o/

### PR DESCRIPTION
**Pull request by Krlier**

### CODE REVIEW

File: server.go [hunk 1]
```diff
@@ -245,6 +245,23 @@ func checkDefaultSecurityTests(configAPI *apiContext.APIConfig) error {
 		return err
 	}
 
+	retirejsQuery := map[string]interface{}{"name": "retirejs"}
+	retirejs, err := analysis.FindOneDBSecurityTest(retirejsQuery)
+	if err == mgo.ErrNotFound {
+		// As Retire JS securityTest is not set into MongoDB, HuskyCI will insert it.
```
Comment (rafaveira3) [inline server.go]: Would mind fixing this typo? `RetireJS`

File: dockers/retirejs/Dockerfile [hunk 2]
```diff
@@ -7,6 +7,7 @@ RUN apk update && apk upgrade \
 	&& apk add --no-cache alpine-sdk bash openssh-client \
 	&& apk add git
 
+RUN npm install
 RUN npm install -g retire
 
 RUN wget -O jq https://github.com/stedolan/jq/releases/download/jq-1.5/jq-linux64
```
Comment (rafaveira3) [inline dockers/retirejs/Dockerfile]: By checking your containers' command in `config.yml` file, I realized that you are not actually using `jq` during your securityTest. What you think about removing  these `jq` lines? 😃

File: analysis/gosec.go [hunk 3]
```diff
@@ -11,12 +11,12 @@ import (
 
 // GosecOutput is the struct that holds issues and stats found on a Gosec scan.
 type GosecOutput struct {
-	Issues []Issue
-	Stats  Stats
+	Issues []GosecIssue
```
Comment (rafaveira3) [inline analysis/gosec.go]: Wow, thanks a lot for this refactoring! Would you mind changing `Issues` to ` GosecIssues` and `Stats` into `GosecStats` as well? 

If so, would also change 
`for _, issue := range gosecOutput.Issues {`  into 
`for _, issue := range gosecOutput.GosecIssues {` ? 😄

Comment (Krlier) [reply analysis/gosec.go]: Sure can! Fixed variable names in 1dc1c00

File: dockers/retirejs/Dockerfile (hunk 2 above)
Comment (Krlier) [reply dockers/retirejs/Dockerfile]: Sure can! Fixed the unnecessary command in 1dc1c00

File: server.go (hunk 1 above)
Comment (Krlier) [reply server.go]: Sure can! Fixed this typo mistake in 1dc1c00

**Pull request merged**